### PR TITLE
detail view with subdetail components

### PR DESCRIPTION
**Pull request by RobDolares**
Related Issue: [9566](https://github.com/open-cluster-management/backlog/issues/9566)

Mock:
<kbd>
Screen Shot 2021-03-17 at 3 14 13 PM
</kbd>

Current Iteration:
<kbd>
Screen Shot 2021-03-17 at 3 16 14 PM
</kbd>

### CODE REVIEW

File: frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx [hunk 1]
```diff
@@ -131,14 +184,84 @@ function DetailsView(props: {
...
-                    {t('policy.report.flyout.back')}
-                </button>
+        <div className={classes.body}>
+            <div className={classes.backAction}>
+                <AngleLeftIcon />
+                <button onClick={() => setDetailsView(false)}>{t('policy.report.flyout.back')}</button>
+            </div>
+            <div className={classes.detailTitleGroup}>
+                <div className={classes.titleText}>{_.get(selectedPolicy, 'results[0].message', '')}</div>
+                <div className={classes.detailActions}>
```
Comment (zlayne) [inline frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx:219]: For the current iteration we don't have any actions to take on the Insight. In the future we will want to be able to press a button and remediate the vulnerability. But for now we can remove the actions dropdown.

File: frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx [hunk 2]
```diff
@@ -131,14 +184,84 @@ function DetailsView(props: {
...
+                        text={t('policy.report.flyout.details.actions')}
+                        onSelect={onSelect}
+                        id="policy-actions"
+                        isKebab={false}
+                        isPrimary={false}
+                    />
+                </div>
+            </div>
+            <div className={classes.detailText}>{_.get(selectedPolicy, 'results[0].data.details', '')}</div>
+            <Link className={classes.knowledgebaseLink} to="/">
```
Comment (zlayne) [inline frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx:240]: id say comment this out until we have a link

**Review by zlayne — APPROVED** [review]
lgtm

File: frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx [hunk 3]
```diff
@@ -49,6 +49,72 @@ const useStyles = makeStyles({
...
+                fontSize: '14px',
+                fontWeight: 700,
+                lineHeight: '18px',
+            },
+            '& svg': {
+                alignSelf: 'center',
+                marginRight: '10px',
+            },
+        },
+    },
```
Comment (showeimer) [inline frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx:117]: Why is there so much custom CSS needed for this design?

Comment (RobDolares) [reply frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx:73]: Planning to cut this down with future design iterations when we are no longer using mock data.

**Review by showeimer — APPROVED** [review]

**Pull request merged**